packages/grafana-ui/src/components/QueryField/QueryField.tsx

```diff
@@ -119,6 +124,16 @@ export class QueryField extends React.PureComponent<QueryFieldProps, QueryFieldS
     }
   }
 
+  onKeyDown = (event: KeyboardEvent, editor: CoreEditor, next: Function) => {
```

Comment:
```suggestion
  onKeyDown = (keyEvent: Event, editor: CoreEditor, next: Function) => {
    const event = keyEvent as KeyboardEvent;
```
Unfortunately I think you need to do something like this to satisfy typechecks